### PR TITLE
[11.0][ADD] sale_stock_secondary_unit: New module for store secondary  unit in stock moves when a sale order is confirmed

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please remove the offending code in the other module.

File: sale_stock_secondary_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    'development_status': 'Beta',
+    'category': 'stock',
+    'website': 'https://github.com/OCA/stock-logistics-warehouse',
+    'author': 'Tecnativa, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'application': False,
+    'installable': True,
+    'depends': [
+        'sale_order_secondary_unit',
+        'product_secondary_unit',
```
Comment (pedrobaeza) [inline sale_stock_secondary_unit/__manifest__.py]: This should be `stock_secondary_unit`

**Comment by sergio-teruel** [comment]
@pedrobaeza @rousseldenis chages done

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**